packages/sui-bundler/webpack.config.prod.js

```diff
@@ -11,6 +11,8 @@ const webpack = require('webpack')
 const definePlugin = require('./shared/define')
 const babelRules = require('./shared/module-rules-babel')
 
+console.log('new bundler')
```

Comment:
A console in production, without emojis? 😱

Reply:
hahaha still, not production ready. :)